src/hooks/useStakingInput.ts

```diff
@@ -1,15 +1,50 @@
+import { BigNumber } from 'ethers';
+
 import { useStakingWidgetContext } from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
 
 import { MotionVote } from '~utils/colonyMotions';
-import { getHandleStakeSuccessFn, getStakingTransformFn } from './helpers';
 
+import { getHandleStakeSuccessFn, getStakingTransformFn } from './helpers';
 import { useAppContext, useColonyContext } from '.';
+import { useGetUserTokenBalanceQuery } from '~gql';
+import { Address } from '~types';
+
+export const useEnoughTokensForStaking = (
```

Comment:
Leaving this here for now since it's not called anywhere at the top level (only within other hooks).